testsUI/spec.js

```diff
@@ -0,0 +1,81 @@
+const Application = require("spectron").Application;
+const assert = require("assert");
+const path = require("path");
+
+describe("Application launch", function() {
+  this.timeout(100000);
+
+  beforeEach(function() {
+    this.app = new Application({
+      // Your electron path can be any binary
+      path: "C:\\Users\\lorenzo\\AppData\\Local\\atom\\atom.exe",
+
+      // Assuming you have the following directory structure
+
+      //  |__ my project
+      //     |__ ...
+      //     |__ main.js
+      //     |__ package.json
+      //     |__ index.html
+      //     |__ ...
+      //     |__ test
+      //        |__ spec.js  <- You are here! ~ Well you should be.
+
+      // The following line tells spectron to look and use the main.js file
+      // and the package.json located 1 level above.
+      args: [path.join(__dirname, "..")],
+    });
+    return this.app.start();
+  });
+
+  afterEach(function() {
+    if (this.app && this.app.isRunning()) {
+      return this.app.stop();
+    }
+  });
+
+  it("shows an initial window", function() {
+    return this.app.client.getWindowCount().then(function(count) {
+      assert.equal(count, 2);
+      // Please note that getWindowCount() will return 2 if `dev tools` are opened.
```

Comment:
Is it mandatory to open the devtools?
The tests should simulate user' view of Molecule, and usually the user doesn't open dev tools.

Reply:
In that case, keep it open yes.